### PR TITLE
add option to pass REGION env; log http code on failure

### PR DESCRIPTION
**Pull request by xueshanf**
Can pass in region as environment variable.
For add and delete, add http return code in case of failure.

Thanks for the great job!

### CODE REVIEW

**Comment by madeddie** [comment]
:+1: the region needs to be configurable

**Comment by ddgenome** [comment]
Thanks for the contribution, it looks good. We are testing it and will likely accept the PR soon. Thanks again!

File: README.md [hunk 1]
```diff
@@ -47,4 +47,6 @@ Usage
 
 ```docker run -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
 
+Or pass in REGION (default is 'us-east-1')
 
+```docker run -e REGION='us-west-2' -v /etc/sysconfig/:/etc/sysconfig/ MonsantoCo/etcd-aws-cluster```
```
Comment (tjcorr) [inline README.md:52]: When I tested I found out that you can't put single quotes around the region value

Comment (madeddie) [inline README.md:52]: More precise, you should never add quotes like you would in a shell script in a systems unit file. The unit file is parsed by systems/fleet and not the shell.

The quotes end up as part of the variable value

**Comment by tjcorr** [comment]
Besides the small issue, this worked for me. Going to merge and correct the documentation afterwards. Sorry I had meant to handle different regions but forgot to. Thanks for the PR!

**Comment by xueshanf** [comment]
Cool, thanks!

**Pull request merged**